File: Python/Lecture1/BubbleSort.py
def bubbleSort(alist):
    for i in range(0, len(alist)):
        for j in range(0, len(alist) - i - 1):
            if (alist[j] > alist[j + 1]):
                temp = alist[j];
                alist[j] = alist[j + 1];
                alist[j + 1] = temp;


alist = [1, 3, 5, 6, 2, 4, 8, 2, 4, 6, 8, 3]
bubbleSort(alist)
print(alist)

File: Lecture1-CSharp/Lecture1/BubbleSort.cs
using System;

namespace Lecture1_CSharp
{
    public class BubbleSort
    {
        public static void Run(int[] a)
        {
            for (int i = 0; i < a.Length; i++)
            {
                for (int j = 0; j < a.Length - i - 1; j++)
                {
                    if (a[j] > a[j + 1])
                    {
                        int temp = a[j];
                        a[j] = a[j + 1];
                        a[j + 1] = temp;
                    }
                }
            }

            foreach (int i in a)
                Console.WriteLine("{0},", i);

        }//end of Main()

    }//end of class BubbleSort

}
